SDDC_FX3/Application.h

```diff
@@ -26,7 +26,10 @@
 #include "cyu3externcstart.h"
 #include "i2cmodule.h"
 
-#define MAXLEN 64		// max buffer debug len
+#include "../Interface.h"
+
+#define TRACESERIAL		/* enable the trace to serial port*/
+#define MAXLEN (100)		// max buffer debug len
```

Comment:
as previous comment, move the constant shared between firmware and PC software into interface.h

Reply:
Thanks!